packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1030,6 +1034,11 @@ class EditableText extends StatefulWidget {
   /// {@end-tool}
   final ValueChanged<String> onSubmitted;
 
+  /// {@template flutter.widgets.editableText.onAppPrivateCommand}
+  /// Called when the result of an app private command is received.
+  /// {@endtemplate}
+  final AppPrivateCommandCallback onAppPrivateCommand;
```

Comment:
Any chance we can have this documentation elaborated? It doesn't really follow our style guide, which requires that documentation be more detailed than could be guessed just from the name. :-)

Reply:
Good call, I'll open a quick PR to elaborate on this.